### DISCUSSION TITLE
Resto v0.1.6

### DISCUSSION THREAD

**Discussion by abdfnx** [abdfnx/resto#21] [Releases]
## What's Changed ?

- add a version checker to check if there's a new release by @abdfnx 
- add `resto settings` command to `Resto CLI` by @abdfnx 
```bash
$ resto settings set show_update true
```
- add settings editor to `Resto UI` and `Resto CLI` by @abdfnx 

## Changelog

* 336dffa docs: add `resto settings` example to root command, add the new shortcuts
* 8e2c449 new feature: add `resto settings` command to `Resto CLI`, update modules
* 5ab51af update: convert boolean objects in `settings` file from string to bool
* d5985cf restore `./scripts/date.go` file
* 44ec8a5 new feature: add resto settings editor for `Resto UI`, fix `ci` CI
* 6a9687b move `build/date.go` to `scripts/date.go` and change all pathes in resto
* e6acc1f add `isUnderAppData` function for windows script installer in `checker`
* 29e95d9 big update: add version checker to resto in `resto cli` and `resto ui`
* 110e29d update resto logo, update `README.md`
* 81b9a10 update resto logo
* 29148c8 remove `#!/usr/bin/env node` from `gh-rs.js` in `scripts` dir
* 472c187 add `--hidden` flag to `install` command flags, update docs

**Full Changelog**: https://github.com/abdfnx/resto/compare/v0.1.5...v0.1.6

<hr /><em>This discussion was created from the release <a href='https://github.com/abdfnx/resto/releases/tag/v0.1.6'>Resto v0.1.6</a>.</em>

**Comment by msvitok77** [comment]
Hello guys. Please how can I use different color themes for resto? I tried to edit settings using ctrl-e changed the color scheme to gruvbox, but no luck. I can see the definitions are present in core/editor/runtime/files/colorschemes. Thanks

**Reply by abdfnx** [reply]
Hi @yshadow, the color schemes are for resto editor not resto itself, thanks for using resto 😊

**Reply by msvitok77** [reply]
aa I see. Ok :-). Maybe an idea for a new feature ;-).
